app/models/user.rb

```diff
@@ -3,4 +3,9 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+
+  validates_presence_of :full_name
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.